### PR TITLE
feat: test power-mon operator on BM

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR tests PowerMon operator on the BM cluster.

### CODE REVIEW

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

File: ci-operator/config/openshift-power-monitoring/power-monitoring-operator/openshift-power-monitoring-power-monitoring-operator-v1alpha1.yaml [hunk 1]
```diff
@@ -55,18 +55,29 @@ tests:
...
+        unset KUBECONFIG
+        echo "Logging to registry"
+        until [ -f /tmp/unsleep ]; do
+            echo "Sleeping..."
+            sleep 60
+        done
+        oc registry login -a /tmp/pullsecret.json
+        echo "exporting the KUBECONFIG"
+        export KUBECONFIG="${OLD_KUBECONFIG}"
         operator-sdk run bundle -n openshift-operators "$OO_BUNDLE"
```
Comment (aleskandro) [inline ci-operator/config/openshift-power-monitoring/power-monitoring-operator/openshift-power-monitoring-power-monitoring-operator-v1alpha1.yaml:81]:
```suggestion
        #!/bin/bash
        sleepUntilUnlocked() {
          until [ -f /tmp/unsleep ]; do
            echo "Sleeping..."
            sleep 60
          done
        }
        HOME=/tmp/home
        XDG_RUNTIME_DIR=/tmp/home/containers
        mkdir -p $XDG_RUNTIME_DIR
        OLD_KUBECONFIG=${KUBECONFIG}
        unset KUBECONFIG
        oc registry login || sleepUntilUnlocked
        echo "exporting the KUBECONFIG"
        export KUBECONFIG="${OLD_KUBECONFIG}"
        operator-sdk run bundle -n openshift-operators "$OO_BUNDLE" || sleepUntilUnlocked
```

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-power-monitoring-operator-v1alpha1-power-monitoring-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Pull request closed without merging**